dynabuffers-python/dynabuffers/ast/FieldType.py

```diff
@@ -17,14 +17,14 @@ def __init__(self, options: FieldTypeOptions):
         self.options = options
 
     def size(self, value, registry):
-        return 1 + len(str(value).encode("utf-8"))
+        return self.options.datatype.size(value, registry)
```

Comment:
Das ist ein Schmankerl - ich würd behaupten, dass das einfach nur zufällig funktioniert hat (weil immer mehr Bytes allokiert wurden als nötig).
Jetzt ist es gleich wie in der Java Version.

Reply:
stimmt, eieiei. :1st_place_medal:  : D